deploy/deploy.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
```

Comment:
Do we need the RBAC resources?

Reply:
I thought rbac was an option? I can provide a sample yaml to create them - might be better to have that in a separate file though

Reply:
This example is unlikely to work on production clusters because they're probably using RBAC.  I'd include it.